src/toil/test/src/jobCacheTest.py

```diff
@@ -702,16 +702,16 @@ def testRemoveLocalImmutablyReadFile(self):
             self._deleteLocallyReadFilesFn(readAsMutable=False)
 
         def _deleteLocallyReadFilesFn(self, readAsMutable):
-            self.options.retryCount = 1
-            A = Job.wrapJobFn(self._writeFileToJobStore, isLocalFile=True)
-            B = Job.wrapJobFn(self._removeReadFileFn, A.rv(), readAsMutable=readAsMutable)
+            self.options.retryCount = 0
+            A = Job.wrapJobFn(self._writeFileToJobStore, isLocalFile=True, memory='10M')
+            B = Job.wrapJobFn(self._removeReadFileFn, A.rv(), readAsMutable=readAsMutable, memory='20M')
             A.addChild(B)
             try:
                 Job.Runner.startToil(A, self.options)
             except FailedJobsException as err:
-                self.assertEqual(err.numberOfFailedJobs, 1)
+                self.assertEqual(err.numberOfFailedJobs, 2)
                 errMsg = self._parseAssertionError(self.options.logFile)
-                if not 'explicitly' in errMsg:
+                if 'explicitly' not in errMsg:
                     self.fail('Shouldn\'t see this')
```

Comment:
No backslashes, please.

'explicitly' is too unspecific and it doesn't help the reader understand what exactly you are expecting for. And "Shouldn't see this" is pretty much useless as far as an assertion error is concerned.